src/main/java/budgetbuddy/storage/loans/JsonSerializableLoansManager.java

```diff
@@ -0,0 +1,56 @@
+package budgetbuddy.storage.loans;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import budgetbuddy.commons.exceptions.IllegalValueException;
+import budgetbuddy.model.LoansManager;
+import budgetbuddy.model.person.Person;
+
+/**
+ * An immutable LoansManager that is serializable to JSON format.
+ */
+@JsonRootName(value = "loansmanager")
+public class JsonSerializableLoansManager {
+
+    public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+
+    private final List<JsonAdaptedPerson> persons = new ArrayList<>();
+
+    /**
+     * Constructs a {@code JsonSerializableLoansManager} with the given persons.
```

Comment:
I'm not sure if it's intended in this class to be about persons rather than loans?

Reply:
Yes it's intentional, as the structure is `LoansManager` -> `UniquePersonsList` .

Reply:
Alright